scripts/test_commands.sh

```diff
@@ -141,3 +163,5 @@ echo
 displayCmd "neu plugins --help"
 neu plugins --help
 echo
+
+rm -rf $tmpDirPath
```

Comment:
Here also, I think you will get a different line when you resolve conflicts ;) :tada: